core/utils/nodes.ts

```diff
@@ -7,6 +18,39 @@ import * as ERR_MSG from '../logger/errors';
 export function getNodeByName(name: string, value: unknown) {
   if (name === 'isNumber') {
     return new IsNumberNode(convertToBool(value as string));
+  }
+  if (name === 'isString') {
+    return new IsStringNode(convertToBool(value as string));
+  } 
+  if (name === 'isBool') {
+    return new IsBoolNode(convertToBool(value as string));
+  } 
+  if (name === 'isUUID') {
+    return new IsUUIDNode(convertToBool(value as string));
+  }
+  if (name === 'isDatetime') {
+    return new IsDatetimeNode(convertToBool(value as string));
+  }
+  if (name === 'equals') {
+    return new EqualsNode(value);
+  }
+  if (name === 'notEquals') {
+    return new NotEqualsNode(value);
+  }
+  if (name === 'min') {
+    return new MinNode(value as number);
+  }
+  if (name === 'max') {
+    return new MaxNode(value as number);
+  }
+  if (name === 'isPositive') {
+    return new IsPositiveNode(convertToBool(value as string));
+  }
+  if (name === 'isNegative') {
+    return new IsNegativeNode(convertToBool(value as string));
+  }
+  if (name === 'isNotEmpty') {
+    return new IsNotEmptyNode(convertToBool(value as string));
```

Comment:
Try to change all that from 'if' to 'else if'. It would be more optimal
```suggestion
  }
  else if (name === 'isString') {
    return new IsStringNode(convertToBool(value as string));
  } 
  else if (name === 'isBool') {
    return new IsBoolNode(convertToBool(value as string));
  } 
  else if (name === 'isUUID') {
    return new IsUUIDNode(convertToBool(value as string));
  }
  else if (name === 'isDatetime') {
    return new IsDatetimeNode(convertToBool(value as string));
  }
  else if (name === 'equals') {
    return new EqualsNode(value);
  }
  else if (name === 'notEquals') {
    return new NotEqualsNode(value);
  }
  else if (name === 'min') {
    return new MinNode(value as number);
  }
  else if (name === 'max') {
    return new MaxNode(value as number);
  }
  else if (name === 'isPositive') {
    return new IsPositiveNode(convertToBool(value as string));
  }
  else if (name === 'isNegative') {
    return new IsNegativeNode(convertToBool(value as string));
  }
  else if (name === 'isNotEmpty') {
    return new IsNotEmptyNode(convertToBool(value as string));
```